GetStream/stream-chat-android
# Fix displaying long usernames in footnote view

#3144 

### 🎯 Goal

Fix displaying long usernames in footnote view

### 🛠 Implementation details

Added a limit and weight to username's number of lines in `FootnoteView`.

### 🎨 UI Changes

_Add relevant screenshots_

| Before | After |
| --- | --- |
| ![Screenshot_1646050926](https://user-images.githubusercontent.com/17440581/155988557-bd9fa481-a617-4654-822e-c99be342a73d.png)|![Screenshot_1646052013](https://user-images.githubusercontent.com/17440581/155988565-4902ed40-6cac-4ca6-b483-109a8c4aa949.png)|


### 🧪 Testing

1. Go to `demo/AppConfig` and change the username's name to something very very long
2. Sign in as this user and send a few messages
3. Sign in as another user and open the channel to which the messages were sent
4. Observe that the footnote is displayed correctly

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
